jams/core.py

```diff
@@ -704,23 +569,48 @@ def to_interval_values(self):
         labels : list
             List view of value field.
         '''
+        ints, vals = [], []
+        for obs in self.obs:
+            ints.append([obs.time, obs.time + obs.duration])
+            vals.append(obs.value)
 
-        times = timedelta_to_float(self.time.values)
-        duration = timedelta_to_float(self.duration.values)
+        return np.array(ints), vals
 
-        return np.vstack([times, times + duration]).T, list(self.value)
+    @property
+    def __json__(self):
+        '''JSON encoding attribute'''
 
-    def __deepcopy__(self, memo):
-        '''Explicit deep-copy implementation'''
-        jf = JamsFrame()
-        for field in self.fields():
-            if len(self[field]):
-                jf[field] = copy.deepcopy(self[field])
-            else:
-                jf[field] = []
+        if self.dense:
+            times, durations, values, confidences = [], [], [], []
+            for (t, d, v, c) in self.obs:
+                times.append(t)
+                durations.append(d)
+                values.append(serialize_obj(v))
+                confidences.append(c)
+
+            return dict(time=times,
+                        duration=durations,
+                        value=values,
+                        confidence=confidences)
+        else:
+            return [dict(time=o.time,
```

Comment:
Some of what?  I mean, we can always do `dict(obs._asdict())`, but that doesn't solve the dict-of-lists/list-of-dicts problem resulting from sparse/dense observation lists.